cinema/views.py

```diff
@@ -167,6 +184,23 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "date"},
+                description="Filter by date (ex. ?date=2022-10-28)"
+            ),
+            OpenApiParameter(
+                "movie",
+                type={"type": "str"},
```

Comment:
type is number

Reply:
Why is that? A string comes in the request query parameters, and inside the filter string changes into the integer